code/modules/vehicle/tank.dm

```diff
@@ -0,0 +1,70 @@
+/obj/vehicle/tank
+	name = "tank"
+	icon = 'icons/vehicles/tanks.dmi'
+	icon_state = "tank_green"
+	vehicle_move_delay = 4
+	var/allow_fire = 0
+	var/recoil_delay = 10
+	var/resume_move = 10
+	var/datum/action/innate/fire_cannon/fire_cannon_action = new
+
+
+/obj/vehicle/tank/New()
+	..()
```

Comment:
Don't define something like this if it does nothing, you're just adding to overhead on creation for no reason at all.